lib/core/client.js

```diff
@@ -504,7 +505,7 @@ class NightwatchClient extends EventEmitter {
       return;
     }
 
-    Object.assign(this.initialCapabilities, props);
```

Comment:
is this necessary?

Reply:
if someone tries to update inner properties, like `'bstack:options': {buildName: 'Some Name'}`

Reply:
reverted